nixos/modules/virtualisation/oci-containers.nix

```
@@ -59,6 +59,18 @@ let
         '';
         };
 
+        environmentFile = mkOption {
+          type = with types; either path (listOf path);
```

```suggestion
        environmentFiles = mkOption {
          type = with types; listOf path;
```

I would suggest this for simplicity.

---

I can do that (and also thought about it before), anyone else who suggests this?

---

@SuperSandro2000 has a good point that it is simple if you just stick to a list... though I will point out that this is your change and you're free to use your judgement on it. Either way is acceptable.

---

Ok I decided for the `environmentFiles` option as it is less ambiguous in naming and simpler.